[examples]/test_vehicles/c_test_gui.lua

```diff
@@ -0,0 +1,71 @@
+local screenW, screenH = guiGetScreenSize()
+
+local windowW, windowH = 300, 200
+local windowX, windowY = (screenW - windowW) / 2, (screenH - windowH) / 2
+local window = guiCreateWindow(windowX, windowY, windowW, windowH, "Vehicle Spawner", false)
+
+local label = guiCreateLabel(20, 40, 260, 30, "Enter Vehicle ID:", false, window)
+guiLabelSetHorizontalAlign(label, "center")
+
+local input = guiCreateEdit(50, 70, 200, 30, "", false, window)
+
+local spawnButton = guiCreateButton(50, 120, 200, 40, "Spawn Vehicle", false, window)
+
+guiSetVisible(window, false)
+
```

Comment:
It doesn't really matter in this case (test script) but it's good practice 😅: could you make the functions local like your variables? `local function fName(...)` or `local fName = function(...)`
Thank you